src/AppInstallerCLICore/Workflows/ShellExecuteInstallerHandler.cpp

```diff
@@ -16,16 +16,15 @@ namespace AppInstaller::CLI::Workflow
         {
             AICLI_LOG(CLI, Info, << "Starting installer. Path: " << filePath);
 
-            SHELLEXECUTEINFOA execInfo = { 0 };
-            execInfo.cbSize = sizeof(SHELLEXECUTEINFO);
+            SHELLEXECUTEINFOW execInfo = { 0 };
+            execInfo.cbSize = sizeof(SHELLEXECUTEINFOW);
             execInfo.fMask = SEE_MASK_NOCLOSEPROCESS;
-            std::string filePathUTF8Str = filePath.u8string();
-            execInfo.lpFile = filePathUTF8Str.c_str();
-            execInfo.lpParameters = args.c_str();
+            execInfo.lpFile = filePath.c_str();
+            execInfo.lpParameters = Utility::ConvertToUTF16(args).c_str();

```

Comment:
>Utility::ConvertToUTF16(args) [](start = 36, length = 29)

This is a temporary whose life ends at the semicolon.  You need to keep it local to be able to pass it in this way.